spec/lib/email_alert_signup_api_spec.rb

```diff
@@ -66,6 +72,23 @@
     end
   end
 
+  context "when a default_frequency is provided" do
+    let(:default_frequency) { "daily" }
+
+    it "returns the url email-alert-api gives back, and appends the default_frequency param" do
+      subscription_url = "http://gov.uk/email/some-subscription"
+      init_simple_email_alert_api subscription_url
```

Comment:
As commented earlier, this would be better with parentheses
```suggestion
      init_simple_email_alert_api(subscription_url)
```